dockstore-client/src/main/java/io/dockstore/client/cli/nested/ApiClientExtended.java

```diff
@@ -236,6 +240,43 @@ public void createBodyPart(MultiPart multiPart, String key, Object formObject) {
                     buildResponseHeaders(response),
                     respBody);
             }
+        } catch (ProcessingException ex) {
+            // This could be caused by a failed Jersey Interceptor/filter, missing message body writers, or other IO exceptions.
+            // Mainly, this error is thrown when the provided WES URL is invalid. For more details, see:
+            // https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ProcessingException.html
+            errorMessage(MessageFormat.format("There was an error processing the HTTP request. This could be caused by an invalid URL: {0}",
```

Comment:
I don't like "This could be caused by an invalid URL". Presumably there are a ton of reasons a 400 might occur (bad descriptor file, invalid or non-accessible data, unsupported language, etc.).

Reply:
I added this after the fact when I typed in a horribly wrong URL and got the error message `value is null` from Jersey. For example:
```
[WES]
url: notAServerAtAll
```
returns a response of:
```
12:35:32.086 [main] ERROR io.dockstore.client.cli.ArgumentUtility - There was an error processing the HTTP request. This could be caused by an invalid URL: value is null
```

From what I saw, the `ProcessingException` is thrown by our local client if there is an error with the HTTP request before it's sent. This means that formatting with the HTTP request itself, such as a bad URL or broken headers should be caught here, but things like a bad descriptor file, bad data, etc. wont be (all responses from the WES server are made into `ApiExceptions`, so if there is a bad descriptor file, that will be caught in the following lines).

I'm not attached to `This could be caused by an invalid URL`, so removed it for now: https://github.com/dockstore/dockstore-cli/pull/116/commits/6a36514bbb030011f56973eb3adbb67c59119ce1. Perhaps error hints like this will be worthwhile including in our user documentation once we have that written up.